File: @LV1/1.7/soylent/soylent.go
package main

import (
	"fmt"
)

func main() {
	n := 0
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		m := 0
		fmt.Scanf("%d", &m)

		rem := 0
		if m%400 != 0 {
			rem = 1
		}

		fmt.Println(m/400 + rem)
	}
}

File: LV1/1.7/soylent/soylent.cpp
#include <cstdio>

int main() {
    int n, m;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%d", &m);
        printf("%d\n", m / 400 + (m % 400 != 0));
    }
}